Sources/TuistKit/Services/Cache/CacheWarmService.swift

```diff
@@ -71,4 +76,17 @@ final class CacheWarmService {
             contentHasher: contentHasher
         )
     }
+
+    private func projectTargets(at path: AbsolutePath, config: Config) throws -> Set<String> {
+        let plugins = try pluginService.loadPlugins(using: config)
+        try manifestLoader.register(plugins: plugins)
+        let projects: [AbsolutePath]
+        if let workspace = try? manifestLoader.loadWorkspace(at: path) {
```

Comment:
I'd probably load the graph and then use `GraphTraverser` to get the project targets

Reply:
if we kept it like this, at the very least I would look into sharing the logic with `FocusService`